src/constans/Constans.ts

```diff
@@ -0,0 +1,14 @@
+export const shortcuts = [
```

Comment:
定数は慣習的にすべて大文字のスネークケースのことが多いですね👀

```suggestion
export const SHORT_CUTS = [
```